scipy/spatial/distance.py

```diff
@@ -1373,6 +1373,12 @@ def dice(u, v, w=None):
     dice : double
         The Dice dissimilarity between 1-D arrays `u` and `v`.
 
+    Notes
+    -----
+    This function computes the Dice dissimilarity index. To compute the 
+    Dice similiarity index, convert one to the other with similarity = 
```

Comment:
typo `similiarity` I think

I think what I'll do here is just fix that typo, flush this through doc-only CI, and if it passes, just merge it so I can clear up the `1.10.0` milestone backlog.

We can just leave the matching issue open until further suggestions from Pamphile and Warren are addressed; this is still a tiny improvement on its own.